dbehailu/team-20-codeu
# fixed the bug with upload image (only works for png/jpg files atm)

There were a ton of merge conflicts that I had to go through and fix, and I didn't want to update the live site while we were prototyping to test it. Should work now though!

To test: 
Just go to your page on https://su19-codeu-20-3625.appspot.com/ and upload a .jpg or .png file with your message and click submit. You should be redirected to your user-page (might not show up immediately there), then you can go to the main feed to see your message + image.
